### ISSUE TITLE
Gru test is failing when Security feature is enabled

### ISSUE THREAD

**Issue by musketyr** [micronaut-projects/micronaut-starter#1495]
### Description

When Security feature is enabled the generated Gru tests fails

### Task List

- [ ] Steps to reproduce provided
- [ ] Stacktrace (if present) provided
- [ ] Example that reproduces the problem uploaded to Github
- [ ] Full description of the issue provided (see below)

### Steps to Reproduce

1. Generate new app with `agorapulse-gru-http` and `security`
2. Run `HelloWorldGruControllerTest` 

 ### Expected Behaviour

The test should pass.

### Actual Behaviour
The test fails with a following assertion error

```
There were 2 errors:
  java.lang.AssertionError(java.lang.AssertionError: Status 401 is not expected. Expected status was 200.)
  org.opentest4j.AssertionFailedError(org.opentest4j.AssertionFailedError: [Response must match file content: gruIndex.json content] JSON documents are different:
Different value found in node "", expected: <{"message":"Hello World"}> but was: <"">.
)
org.gradle.internal.exceptions.DefaultMultiCauseException: There were 2 errors:
  java.lang.AssertionError(java.lang.AssertionError: Status 401 is not expected. Expected status was 200.)
  org.opentest4j.AssertionFailedError(org.opentest4j.AssertionFailedError: [Response must match file content: gruIndex.json content] JSON documents are different:
Different value found in node "", expected: <{"message":"Hello World"}> but was: <"">.
)
```

### Environment Information

- **Operating System**: TODO
- **Micronaut Version:** TODO
- **JDK Version:** TODO

### Example Application

- TODO: link to github repository with example that reproduces the issue

**Comment by musketyr** [comment]
I will work on this after the GeeCON conference

**Comment by ojebari-ma** [comment]
Can you please confirm that this is still an issue in the latest version of Micronaut?

**Comment by musketyr** [comment]
not really, because the community features from Agorapulse has been removed in v4. do you know why? can they be added back?

**Comment by sdelamo** [comment]
@musketyr we disabled community features when we release Micronaut 4 because we needed those third party libraries to be compiled with Micronaut 4. Are they now Micronaut 4 compatible?

**Comment by musketyr** [comment]
Hi, yes, the latest versions are now compatible with 4.x.
